lib/python/rose/suite_engine_procs/cylc.py

```diff
@@ -75,6 +75,14 @@ def check_global_conf_compat(self):
                     "[hosts][localhost]work directory"]:
             out = self.popen("cylc", "get-global-config", "-i", key)[0]
             lines = out.splitlines()
+            try:
+                lines = lines.decode()
```

Comment:
`lines` should always be a list so `lines.decode` should always raise `AttributeError`.